mechanical/mechanical_integration/README.md

```diff
@@ -1,32 +1,166 @@
+</tr>
+</table>
+ 
+**Figure 4: Attaching the corner steering assembly**
+
+### 1.2.2 Attach Remaining Assemblies
+Repeat the steps above to mount the remaining corner steering assemblies to the other rocker bogie
+   
+## 1.3 Attaching Corner Wheels
+
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/step_3.png"></td>
+	<td><img src="../../images/mechanical_integration/step_4.png"></td>
+</tr>
+</table>
+
+
+**Figure 5: All corner steering assemblies attached**
+
+Attach the four corner wheel assemblies to the corner steering assemblies. They are all symmetric. You may need to spin the encoder gears to mesh with the motor shaft gears if they are not aligned initially. See Figure 6
+assembly, again making sure to select the encoder mounts such that the encoders wind up on the outside of the rover.
+
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/step_5.png"></td>
+	<td><img src="../../images/mechanical_integration/step_6.png"></td>
+</tr>
+</table>
+
+**Figure 6: Attaching the wheels**
+ 
+
+## 1.4 Mounting Body to Rocker-Bogie
+
+
+
+
+
+### 1.4.1 Attach Rocker-Bogie Legs to Body
+Mount the main body of the rover to the two separate rocker-bogie legs using 15 inch aluminum rod S16A and collar clamps **S22** as shown in Figure 7. Make sure to maintain the order of the collar clamps shown in Figure 7 to fully lock all assemblies along the rod. The rod will go through the pillow blocks on each rocker-bogie leg, and it should pass through the 4th from the back and 2nd up hole on the body as shown in Figure 7.
+
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/step_7.png"></td>
+	<td><img src="../../images/mechanical_integration/step_8.png"></td>
+</tr>
+</table>
+
+**Figure 7: Mounting Body to Rocker-Bogie**
+
+## 1.5 Attach the Differential Pivot
+
+**Table 2: Parts/Tools Necessar**y
+
+|Item |Ref |Qty |Image|
+|-----|----|----|-----|
+|0.5” Hollow aluminum tube 2” long|S18|1|<img src="../../images/components/structural/S18.png" width="30%">|
+|0.5” Collar clamp|S22|1| <img src="../../images/components/structural/S22.png" width="30%">|
+|0.5” Nylon Washer |W3 |5| <img src="../../images/components/washers/W3.png" width="30%">
+
+
+### 1.5.1 Mounting the differential pivot center
+Mount a collar clamp **S22** onto the end of the 2” Aluminum Rod **S18** as well as two of the 0.5” Nylon washers **W3** as shown in Figure 8. Then, insert this rod into the pillow bearing blocks from the inside of the body assembly. Add three more nylon washers on the outside / top and then attach the clamping assembly on the middle of the differential pivot (see Figure 8).
+
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/diff_step_6b.png"></td>
+	<td><img src="../../images/mechanical_integration/diff_step_7.png"></td>
+</tr>
+</table>
+
+**Figure 8: Inserting differential pivot joint**
+
+   
+###1.5.2 Attaching the differential pivot arms
```

Comment:
```suggestion
### 1.5.2 Attaching the differential pivot arms
```
It doesn't render well without the space.
You can just hit 'commit' to include this proposed suggestion so you don't actually have to go back and commit manually

Reply:
done